zeppelin-web/src/app/interpreter/interpreter.controller.js

```diff
@@ -85,7 +85,10 @@ angular.module('zeppelinWebApp').controller('InterpreterCtrl',
           $scope.suggestions = [];
         }
       }
-      if (event.keyCode < 37 || event.keyCode > 40) { // arrow buttons.
+      if (event.keyCode === 13) {
+        closeAutoComplete();
+      }
+      else if (event.keyCode < 37 || event.keyCode > 40) { // arrow buttons.
```

Comment:
Can we move this line to the above line? As google code style said [https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style](https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style).